docs/installation.md

```diff
@@ -6,17 +6,26 @@ layout: page
 # Manual installation
 
 
-### Requirements
+## Downloading
 
-You should have a reasonably new kernel (4.0+), with kernel modules `hid-logitech-dj`
+Clone solaar from GitHub via `git clone https://github.com/pwr-Solaar/Solaar.git`
+
+
+## Requirements for running Solaar
+
+Installing Solaar from a repository should have set up all these requirements
+so in this situation you should be able to skip this section.
+
+Solaar needs a reasonably new kernel (4.0+), with kernel modules `hid-logitech-dj`
```

Comment:
I am not sure about the accuracy of this anymore. I think it might not work before 5.0.

Reply:
Updating this to 5.0 seems like a good idea.  Will do.